### ISSUE TITLE
[Feature Request] Percentage instead of icon.

### ISSUE THREAD

**Issue by f3bruary** [JarekSed/batterymon-clone#4]
Could you have it show the percentage level instead of an icon ? Optionally colored (>20 green/white <20 red). I'm used to tint2's built in battery monitor myself and I got one on my phone too. Much more informative to look at than an icon imo. Anyways it's just a request.

**Comment by JarekSed** [comment]
Good idea. Should be a very small change, I'll hopefully have time this weekend. (Or, feel free to send a pull request)
